src/interactive/repl.ts

```diff
@@ -401,7 +401,7 @@ async function executeCodeCopyPaste(text, individualLine) {
         text = text + '\n'
     }
 
-    await startREPL(true)
+    await startREPL(true, false)
```

Comment:
This should be `true` since we need to look into REPL for this case ?

Reply:
True.